### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Gioman101**

### CODE REVIEW

**Comment by Gioman101** [comment]
Adding a few more Repo's

File: README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@
 - [`FlipperZeroSub-GHz` Debruin & vehicle keyfob dumps.](https://github.com/MuddledBox/FlipperZeroSub-GHz)
 - [`BadUSB Payloads` Generic Duckyscript payloads.](https://github.com/hak5/usbrubberducky-payloads)
 - [`csv2ir` Full conversion of the IRDB repo to Flipper's format.](https://github.com/Spexivus/csv2ir)
+- [`FlipperAmiibo` Bank vault of Amiibos to Flipper's format.](https://github.com/Gioman101/FlipperAmiibo)
```
Comment (djsime1) [inline README.md:27]: I also have my own collection of Amiibo's, perhaps we should compare before listing the repo.

File: README.md [hunk 2]
```diff
@@ -24,6 +24,8 @@
...
+- [`BadUSB WiFi grabber` grabs saved Wi-Fi info & sends the info to email.](https://github.com/Gioman101/BadUSBwifi-grabberFlipper)
```
Comment (djsime1) [inline README.md:28]: There's nothing Flipper specific about this, plus it's just a single file.

**Comment by djsime1** [comment]
Not going to add the WiFi grabber since it would fit better in a Duckyscript repo than here. I link hak5's repo because it's a good starting point. As for Amibo's, I'll probably merge that as-is.

**Comment by djsime1** [comment]
Reopening so I can mark this as merged.

**Pull request merged**